python/taichi/type/annotations.py

```diff
@@ -6,6 +6,8 @@ class ArgAnyArray:
 
     Args:
         element_dim (Union[Int, NoneType], optional): None if not specified (will be treated as 0 for external arrays), 0 if scalar elements, 1 if vector elements, and 2 if matrix elements.
+        element_shapes (Union[Tuple[Int], NoneType]): None if not specified, shapes of each element. For example, element_shapes must be 1d for vector and 2d tuple for matrix.
```

Comment:
re variable name here: do we prefer `element_shapes` or `element_shape` here? @k-ye

Reply:
Let's go with `element_shape`